src/dotnet/ExecutablePackageObtainer/ExecutablePackageObtainer.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.ExecutablePackageObtainer
```

Comment:
I think ToolPackageObtainer makes more sense as a name here. What do you think?

Reply:
ToolPackageObtainer, "tools" is a feature name, that is too board. I think ExecutablePackage is more close to what this model actually does

Reply:
When will this ever be used for anything else, as an executable, besides for tools? I think this communicates the intended usage of these APIs much more clearly.

Reply:
renamed